docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
+
+## Testing your credentials
+
+Before moving to REANA, check that your credentials are in order.
+
+```console
+# Login to lxplus
+$ ssh johndoe@lxplus.cern.ch
+# Make sure userkey.pem is read/write only by the owner
```

Comment:
What about a separation empty line here as you have below? It makes the snippet more readable.